README.md

```diff
@@ -265,7 +267,7 @@ __Parameters__
 |-------------|------------------------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `data`      | `Dict[str, Union[float, int]]` | `None`  | Dictionary of metrics to log. Each metric value is associated with a step. To log multiple metrics at once, pass multiple key-value pairs.                                                                                                                           |
 | `step`      | `Union[float, int]`           | `None`  | Index of the log entry. Must be increasing. <br> **Tip:** Using float rather than int values can be useful, for example, when logging substeps in a batch. |
-| `timestamp` | `datetime`, optional                     | `None`  | Time of logging the metadata.                                                                                                                                                                                                                                        |
+| `timestamp` | `datetime`, optional                     | `None`  | Time of logging the metadata. If not provided, the current time is used. If provided, and `timestamp.tzinfo` is not set, the time is assumed to be in the local timezone.                                                                                            |
```

Comment:
@normandy7 
>  If not provided, the current time is used

The default should be current time in this case

> If provided, and `timestamp.tzinfo` is not set, the time is assumed to be in the local timezone.

Isn't this true for `Run().creation_time` as well?